src/hw6/Flower.java

```diff
@@ -0,0 +1,4 @@
+package hw6;
+
+public class Flower {
```

Comment:
In this class should be field price or cost, a getter, a constructor with parameters and maybe this class should be abstract.
What do you think?

Reply:
everything fixed